pypeit/coadd1d.py

```diff
@@ -112,7 +112,20 @@ def load_arrays(self):
                 if 'RA' in sobjs[indx][0].keys() and 'DEC' in sobjs[indx][0].keys():
                     header_out['RA_OBJ']  = sobjs[indx][0]['RA']
                     header_out['DEC_OBJ'] = sobjs[indx][0]['DEC']
-
+            # Check if the arrays need to be padded
```

Comment:
I'm fine with leaving this in, but it will eventually be solved with lists. Maybe please just add a todo here indicating that a better way is to use lists rather than arrays and add a TODO for the future so we remember to remove this.

Reply:
Sounds good - Added TODOs and also added an issue as a reminder to address this in the future: #1575